include/device.hpp

```diff
@@ -271,9 +271,9 @@ namespace D3D11On12
         }
 
         // Returns true if synchronization was successful, false likely means device is removed
-        bool WaitForCompletion(D3D12TranslationLayer::COMMAND_LIST_TYPE commandListType) noexcept
+        bool WaitForCompletion(D3D12TranslationLayer::COMMAND_LIST_TYPE commandListType) // Can't be marked as noexcept as it throws

```

Comment:
Same as in the other review, I'm not a fan of these "// can't be" comments personally.